docs/network-policy.md

```diff
@@ -236,8 +242,83 @@ ephemeral and unpredictable.
 - Rules assume the priority in which they are written. i.e. rule set at top
   takes precedence over a rule set below it.
 
+## Antrea NetworkPolicy
+
+Antrea NetworkPolicy is another Policy CRD, which is similar to the
+ClusterNetworkPolicy CRD, however its scope is limited to a Namespace.
+The purpose of introducing this CRD is to allow admins to take advantage of
+advanced NetworkPolicy features and apply them within a Namespace to
+complement the K8s NetworkPolicies. Similar to the ClusterNetworkPolicy
+resource, Antrea NetworkPolicy can also be associated with Tiers.
+
+**Note**: Antrea NetworkPolicy is currently in "Alpha" stage and is enabled
+along with Tiers and ClusterNetworkPolicy as part of the `AntreaPolicy`
+feature gate.
+
+### The Antrea NetworkPolicy resource
+
+An example Antrea NetworkPolicy might look like this:
+```yaml
+apiVersion: security.antrea.tanzu.vmware.com/v1alpha1
+kind: NetworkPolicy
+metadata:
+  name: test-anp
+  namespace: default
+spec:
+    priority: 5
+    tier: SecurityOps
+    appliedTo:
+      - podSelector:
+          matchLabels:
+            role: db
+    ingress:
+      - action: Allow
+        from:
+          - podSelector:
+              matchLabels:
+                role: frontend
+          - podSelector:
+              matchLabels:
+                role: nondb
+            namespaceSelector:
+              matchLabels:
+                role: db
+        ports:
+          - protocol: TCP
+            port: 8080
+    egress:
+      - action: Drop
+        to:
+          - ipBlock:
+              cidr: 10.0.10.0/24
+        ports:
+          - protocol: TCP
+            port: 5978
+```
+
+### Policy ordering based on priorities
```

Comment:
This section should be moved to ClusterNetworkPolicy as it applies to both?

Reply:
are you suggesting to move this out one level in the following way:
- [Antrea NetworkPolicy](#antrea-networkpolicy)
  - [The Antrea NetworkPolicy resource](#the-antrea-networkpolicy-resource)
  - [Key differences from ClusterNetworkPolicy](#key-differences-from-clusternetworkpolicy)
- [Policy ordering based on priorities](#policy-ordering-based-on-priorities)
- [Notes](#notes)

Reply:
I think we either move common contents to a separate common section, or put them in ClusterNetworkPolicy (as it is talked first). I saw sections like "Rule enforcement based on priorities" are similar to this one?

Reply:
moved to CNP section ..